azkaban-webserver/src/main/resources/azkaban/webapp/servlet/velocity/executingflowpage.vm

```diff
@@ -112,6 +112,9 @@
         <li class="nav-button pull-right"><button type="button" id="cancelbtn" class="btn btn-danger btn-sm">Kill</button></li>
         <li class="nav-button pull-right"><button type="button" id="retrybtn" class="btn btn-success btn-sm">Retry Failed</button></li>
         <li class="nav-button pull-right"><button type="button" id="executebtn" class="btn btn-success btn-sm">Prepare Execution</button></li>
+        #if ($executionExternalLinkURL)
+          <li class="nav-button pull-right"><a id="analyzerButton" href="${executionExternalLinkURL}" class="btn btn-info btn-sm" type="button" target="_blank" title="Analyze job in ${executionExternalLinkLabel}" tabindex="-1">${executionExternalLinkLabel}</a></li>
```

Comment:
Sorry, I am not sure I follow the "the tab order is broken" part. Could you elaborate a bit more?

I thought the resume and pause buttons wouldn't be displayed at the same time. They are controlled by the logic in the js.

In any case, could you add comment in the code to explain? It may not be obvious to the readers of this code.